### ISSUE TITLE
number of epoch

### ISSUE THREAD

**Issue by zhanghongya0727** [thunlp/ERNIE#75]
Hello, according to your code and experimental Settings, I have also reported the F1 value in the downstream task FEWREL, which is about 1.5% lower than the pre-trained language model you published. Could you please tell me whether the pre-trained language model you published uses one epoch or three epochs?

**Comment by zzy14** [comment]
Hi! It depends on the random seed. You can try to run more epochs to get more stable results, such as 10 epochs.
